contracts/gateway/EIP20CoGateway.sol

```diff
@@ -925,6 +925,21 @@ contract EIP20CoGateway is GatewayBase {
         );
     }
 
+    /**
+     * @notice Gets the penalty amount.
```

Comment:
Please add documentation stating, it will return zero in cases of message doesn't exist or status is progressed/revoked.